### DISCUSSION TITLE
How do I hide page content while viewing a section?

### DISCUSSION THREAD

**Discussion by ghost** [willfaught/paige#49] [Q&A]
Clicking on a link in the menu navigation menu lists posts with all information the front matter, including the content.

How can I disable the actual content from being displayed when viewing a list?

**Comment by willfaught** [comment]
For anyone else looking for an answer: If you insist on a list page having content, but not showing it, you can either hide it with styling, or override the list template to omit it. See the `paige.style` site or page parameter. I think the simplest thing is to just delete the content, though.
